controllers/perses_controller_test.go

```diff
@@ -36,6 +36,7 @@ var _ = Describe("Perses controller", func() {
 		typeNamespaceName := types.NamespacedName{Name: PersesName, Namespace: PersesName}
 		configMapNamespaceName := types.NamespacedName{Name: common.GetConfigName(PersesName), Namespace: PersesName}
 		persesImage := "perses-dev.io/perses:test"
```

Comment:
perhaps the tag of the persesImage should be updated ? even the image name I guess that's not accurate.

Reply:
yes, is not accurate as is set for the testing case. This checks that the default image is not used and the custom image is set.